### PR TITLE
Development

### PR DESCRIPTION
**Pull request by mbstacy**
Release tag initial tested out but removed based on team input.

**Image tag** 
- Default will use target_stack based on branch
- Override: set `target_image_tag` in k8s_vars file

 **EKS Cluster Config**

- Use tool to set EKS CIDRs for github action runner

### CODE REVIEW

File: src/k8sdeploy/k8sdeploy.py [hunk 1]
```diff
@@ -22,9 +24,13 @@ def __init__(self,var_filename,stack,ecr_account_id):
     def __deploy_data__(self,filename,ecr_account_id):
         self.checkAWSToken(ecr_account_id)
         var_data=self.read_variable_file(filename)
+        # manual inputs
+        var_data['ecr_account_id'] = ecr_account_id
+        var_data['target_stack']=self.stack
+        #var_data['target_release_tag'] = self.release_tag[1:] if self.release_tag.lower().startswith('v') else self.release_tag
```
Comment (michaelkerry) [inline src/k8sdeploy/k8sdeploy.py:30]: in general I think it's best to remove commented-out code - we can always recover it from history if necessary

File: src/k8sdeploy/k8sdeploy.py [hunk 2]
```diff
@@ -22,9 +24,13 @@ def __init__(self,var_filename,stack,ecr_account_id):
...
+        # secrets and config map vars
         secret_cm=self.generate_secret_configmap_data(var_data)
         data = secret_cm | var_data
-        data['ecr_account_id'] = ecr_account_id
         data = self.load_defaults('default_vars.yml',data) | data 
         #data = self.read_variable_file('k8s_vars/default_vars.yml') | data
```
Comment (michaelkerry) [inline src/k8sdeploy/k8sdeploy.py:35]: ibid

File: src/k8sdeploy/k8sdeploy.py [hunk 3]
```diff
@@ -196,4 +202,48 @@ def deploy_objects(self,action="apply",delete_namespace=False):
...
+        self.cluster_name=f"adexk8s-eks-cluster-{self.stack}"
+        self.ip4 = github_runner_ip
+        self.eks=boto3.client('eks')
+
+    @contextmanager
+    def disable_exception_traceback(self):
+        """
+        All traceback information is suppressed and only the exception type and value are printed
+        """
+        default_value = getattr(sys, "tracebacklimit", 1000)  # `1000` is a Python's default value
```
Comment (michaelkerry) [inline src/k8sdeploy/k8sdeploy.py:220]: what does this default value mean, practically speaking?

**Review by michaelkerry — CHANGES_REQUESTED** [review]
couple small things...

**Review by michaelkerry — APPROVED** [review]

File: src/k8sdeploy/k8sdeploy.py (hunk 3 above)
Comment (mbstacy) [reply src/k8sdeploy/k8sdeploy.py:216]: default_value is the length of the trackeback. I use this when a command-line tool: removes traceback and gives functional information to the user of the tool.

**Review by robruma — APPROVED** [review]

**Pull request merged**